src/sinks/util/buffer/metrics.rs

```diff
@@ -334,7 +334,7 @@ mod test {
         (buffered, sent_requests)
     }
 
-    #[tokio::test(core_threads = 2)]
+    #[test_util::test]
```

Comment:
I'm pretty sure this `core_threads` specification is important

Reply:
I did not checked `tokio::test` deeply when initially modified this test, `tokio::test(core_threads = 2)` worked because this change basic_scheduler to threaded. Yesterday when explored code I realized that `tokio::test(threaded_scheduler)` will also work.
In our macros threaded_scheduler by default, so we ok with `test_util::test`.